src/index.html

```diff
@@ -17,6 +30,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1>MOYO HEADER</h1>
+    <header>
+      <a
+        class="logo"
+        href="#logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo MOYO"
+        />
+      </a>
+
+      <nav>
+        <ul>
+          <li>
+            <a
+              href="#apple"
+              class="link is-active first-box"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
```

Comment:
[checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md#html-formatting): Add empty lines between multiline sibling blocks of HTML